content/brazilian-portuguese/java-basics/activity-1.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Imprimir declarações e comentários"
+description: "Introduzir métodos println() e comentários em Java."
+date: 2021-10-13T00:00:00Z
+weight: 2
+---
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/uYoq-4juYBY" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
```

Comment:
You can delete the video iframes for now since they are in English - we can readd videos once they are recreated in Portuguese

```suggestion
```